soc/nxp/imxrt/soc.yml

```diff
@@ -34,3 +34,4 @@ family:
     - name: mimxrt685s
       cpuclusters:
       - name: cm33
+      - name: adsp
```

Comment:
For the RT595, we name the ADSP core `f1`, since it is a fusion F1. My rational for doing this is that it follows how we name ARM core clusters in tree. Would you prefer to keep the `adsp` naming here, or refer to this DSP as the `hifi4`? I don't have a strong preference